docs/concepts/Dependency-Resolution.md

```diff
@@ -49,16 +49,16 @@ When an application specifies an exact version number, such as 1.2, that is not
 
 <a name="floating-versions"></a>
 
-#### Floating (wildcard) versions
+#### Floating versions
 
-A floating or wildcard dependency version is specified with the \* wildcard, as with 6.0.\*. This version specification says "use the latest 6.0.x version"; 4.\* means "use the latest 4.x version." Using a wildcard allows a dependency package to continue evolving without requiring a change to the consuming application (or package).
+A floating dependency version is specified with the \* character. For example, `6.0.*`. This version specification says "use the latest 6.0.x version"; `4.*` means "use the latest 4.x version." Using a floating version reduces changes to the project file, while keeping up to date with the latest version of a dependency.
```

Comment:
If emphasize or capitalization is changed by my other feedback, also change it here for "floating version".